bugbug/models/spambug.py

```diff
@@ -35,6 +35,9 @@ def __init__(self, lemmatization=False):
             bug_features.component(),
             bug_features.num_words_title(),
             bug_features.num_words_comments(),
+            bug_features.keywords(),
+            bug_features.product(),
+            bug_features.component(),
```

Comment:
Oops, product and component were already there :)

Reply:
Ah, missed that. Will add only keywords then.